api/AudioBufferSourceNode.json

```diff
@@ -445,15 +443,13 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBufferSourceNode/start",
           "support": {
             "chrome": {
-              "version_added": "14",
-              "version_removed": "57"
+              "version_added": "24"
```

Comment:
What was the test for this? The AudioBufferSourceNode interface wasn't always exposed, and it seems weird to have this interface at all without the start() method, so what happened in Chrome 24? https://github.com/mdn/browser-compat-data/issues/6656 also makes me suspect tests could get this wrong.

Since this is messy, splitting it out from this PR and handling separately might be good.

Reply:
The test code was as follows:

```javascript
(function() {
  var audioCtx = new (window.AudioContext || window.webkitAudioContext)();
  if (!audioCtx) {return false};
  var instance = audioCtx.createBufferSource();
  return 'start' in instance;
})();
```

Interestingly enough, running it in Chrome 15 and Chrome 23 _do_ return `false`, and Chrome 24 returns `true`.

The original data came from Confluence, it appears -- see #1346.

Reply:
OK, so you did have an instance here, the test should be reliable. I tried it in http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8434 and also observed first support in Chrome 24.

It looks like https://trac.webkit.org/changeset/129260/webkit (WebKit 537.12.0) is when `start` and `stop` were added to `AudioBufferSourceNode` and `OscillatorNode`, then called just `Oscillator` in WebKit. That matches this being in Chrome 24.

To complete this, can you make the same change to `OscillatorNode`? (That has both `start` and `stop`, while `stop` is oddly missing here...)

Reply:
Going to https://developer.mozilla.org/docs/Web/API/AudioBufferSourceNode/stop seems to redirect to `AudioScheduledSourceNode` instead.  To limit the scope of this PR, I will cherry-pick these changes into their own PR, removing `AudioBufferSourceNode.start` and just keeping `AudioScheduledSourceNode.start`.